src/components/Vault/VaultRow.tsx

```diff
@@ -53,7 +53,7 @@ const VaultRow: FC<VaultProps> = (props) => {
             <VaultStrategyButton {...props} />
           </TableCell>
           <TableCell className="pointer-events-none text-center">
-            <VaultApr {...props} />
+            <VaultApy {...props} />
```

Comment:
I think you forgot to do this in your previous PR @vanillaHill ?

Reply:
yes, thx